### PR TITLE
Fix horizontal-overflow test

### PR DESCRIPTION
**Pull request by jatzz10**
Screenshot 2020-08-27 at 2 30 04 PM

### CODE REVIEW

File: ccc/test_credit_card.py [hunk 1]
```diff
@@ -19,6 +19,10 @@ def browser(module_browser, base_url, request):
     time.sleep(4)
     return module_browser
 
+@pytest.mark.parametrize('browser', [('desktop_1'), ('mobile_1')], indirect=True)
```
Comment (ShihaMohan235) [inline ccc/test_credit_card.py:22]: do we have to keep it at start ?

Comment (jatzz10) [reply ccc/test_credit_card.py:22]: Yes, right now, if we run another test before this overflow test, this test fails. So, that's why have placed it at the top.

File: ccc/test_credit_card.py [hunk 2]
```diff
@@ -19,6 +19,10 @@ def browser(module_browser, base_url, request):
...
+def test_overflowing(browser):
```
Comment (aikanshgarg) [inline ccc/test_credit_card.py:23]: overflow would be better imo

Comment (jatzz10) [reply ccc/test_credit_card.py:23]: Right, making this change now.

File: ccc/test_credit_card.py (hunk 1 above)
Comment (ShihaMohan235) [reply ccc/test_credit_card.py:22]: why is that ?
